backend/api/Startup.cs

```diff
@@ -138,6 +141,20 @@ public void ConfigureServices(IServiceCollection services)
             services.AddScoped<IKeycloakRequestClient, KeycloakRequestClient>();
 
             services.AddAutoMapper(typeof(Startup));
+
+            services.AddHealthChecks()
+                .AddCheck("liveliness", () => HealthCheckResult.Healthy())
+                .AddSqlServer(builder.ConnectionString, tags: new[] { "services" });
+            //TODO: Add a health check for keycloak connectivity.
+            services.AddHealthChecksUI(setupSettings: setup =>
+            {
+                setup.AddHealthCheckEndpoint("liveliness", "http://localhost/live");
+                setup.AddHealthCheckEndpoint("readiness", "http://localhost/ready");
+                setup.AddWebhookNotification("rocketchat",
+                    uri: Configuration["ROCKETCHAT_HEALTH_HOOK"],
```

Comment:
These are stored as secrets. The two values below could also be hardcoded in theory, but I like knowing which env the healthchecks are failing in.

Reply:
When does this get called/triggered?

Reply:
Triggered every 10 minutes when a health check is failing (configurable) or once when a failing health check stops failing.